test/config-network/config-network-http-https.yaml

```diff
@@ -0,0 +1,140 @@
+# Copyright 2019 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-network
+  namespace: knative-serving
+  labels:
+    serving.knative.dev/release: devel
+
+data:
+  httpProtocol: "Enabled"
+  autoTLS: "Enabled"
```

Comment:
We cannot turn on auto TLS in our E2E tests until 1) either we set up DNS servers for the real domain to get real CA signed TLS certificates or 2) we use CA issuers https://github.com/knative/serving/issues/4066#issuecomment-521064350 to get self-signed certificates.

At this point, in order to test HTTPS enabled, we may need to manually provide a self-signed certificate by using openssl and configure Gateway to support TLS.